docs/concepts/validation_decorator.md

```diff
@@ -319,16 +244,19 @@ asyncio.run(main())
 # requires: `conn.execute()` that will return `'testing@example.com'`
 ```
 
-## Custom config
+## Compatibility with type checkers
+
+As the [`validate_call()`][pydantic.validate_call] decorator preserves the decorated function's signature,
+it should be compatible with type checkers. However, due to current limitations in the Python type system,
+the [`raw_function`](#accessing-the-original-function) or other attributes won't be recognized and you will
+need to suppress the error.
```

Comment:
Might be helpful to use some of the original copy here - lots of folks are familiar with `mypy` specifically, and we want to specifically reference how to suppress the error:

```
The `validate_call` decorator should work "out of the box" with [mypy](http://mypy-lang.org/) since it's
defined to return a function with the same signature as the function it decorates. The only limitation is that
since we trick mypy into thinking the function returned by the decorator is the same as the function being
decorated; access to the [raw function](#raw-function) or other attributes will require `type: ignore`.
```